<!DOCTYPE html>
<html>

<head lang="en">
    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
    <meta charset="utf-8">
    <title>Tao Wang Lab | Bioinformatics and Computational Biology Department |
        MD Anderson Cancer Center</title>
    <meta name="viewport"
        content="width=device-width, initial-scale=1, maximum-scale=1">
    <meta name="author" content="Fangjiang Wu">
    <meta name="description"
        content="MD Anderson Cancer Center, Bioinformatics and Computational Biology Department, Tao Wang Lab" />
    <meta name="keyword"
        content="MD Anderson Cancer Center, Tao Wang Lab, Bioinformatics and Computational Biology Department" />

    <!--Favicon-->
    <link rel="shortcut icon" href="images/favicon.ico" title="Favicon" />

    <!-- Main CSS Files -->
    <link rel="stylesheet" href="css/style.css">

    <!-- Namari Color CSS -->
    <link rel="stylesheet" href="css/namari-color.css">

    <!--Icon Fonts - Font Awesome Icons-->
    <link rel="stylesheet" href="css/font-awesome.min.css">

    <!-- Animate CSS-->
    <link href="css/animate.css" rel="stylesheet" type="text/css">

    <!-- Main CSS Files -->
    <link rel="stylesheet" href="vendor/bootstrap/css/bootstrap.min.css">

    <link href="css/custom.css" rel="stylesheet" type="text/css">

    <!--Google Webfonts-->
    <link
        href='https://fonts.googleapis.com/css?family=Open+Sans:400,300,600,700,800'
        rel='stylesheet' type='text/css'>
</head>

<body>
    <!-- Preloader -->
    <div id="preloader">
        <div id="status" class="la-ball-triangle-path">
            <div></div>
            <div></div>
            <div></div>
        </div>
    </div>
    <!--End of Preloader-->

    <div class="page-border" data-wow-duration="0.7s" data-wow-delay="0.2s">
        <div class="top-border wow fadeInDown animated"
            style="visibility: visible; animation-name: fadeInDown;"></div>
        <div class="right-border wow fadeInRight animated"
            style="visibility: visible; animation-name: fadeInRight;"></div>
        <div class="bottom-border wow fadeInUp animated"
            style="visibility: visible; animation-name: fadeInUp;"></div>
        <div class="left-border wow fadeInLeft animated"
            style="visibility: visible; animation-name: fadeInLeft;"></div>
    </div>

    <div id="wrapper">

        <div id="header" class="nav-collapse">
            <div class="container clearfix" style="padding: 0 30px">
                <div>
                    <!-- <div id="logo">
                            WANG LAB
                        </div> -->
                    <!--Main Navigation-->
                    <nav id="nav-main">
                        <ul>
                            <li>
                                <a href="index.html">Home</a>
                            </li>
                            <li>
                                <a href="research.html">Research</a>
                            </li>
                            <li>
                                <a href="product.html">Product</a>
                            </li>
                            <li>
                                <a href="software.html">Software</a>
                            </li>
                            <li>
                                <a href="talk.html">Talks</a>
                            </li>
                            <li>
                                <a href="member.html">Members</a>
                            </li>
                            <!-- <li>
                                <a href="aboutPI.html">About PI</a>
                            </li> -->
                        </ul>
                    </nav>
                    <!--End of Main Navigation-->

                    <div id="nav-trigger"><span></span></div>
                    <nav id="nav-mobile"></nav>

                </div>
            </div>
        </div>

        <!--Main Content Area-->
        <main id="content">
            <section class="bg bg-long">
                <div class="container">
                    <div class="section-heading text-center"
                        style="margin-top: 80px">
                        <h2 class="section-title">Software Packages</h2>
                    </div>
                    <div class="row">
                        <div class="col-md-12 col-sm-12 col-xs-12">
                            <div class="project">
                                <div class="row">
                                    <div class="col-md-4 col-sm-12">
                                        <img src="images/software/toolbox.jpg"
                                            alt="" class="img-responsive">
                                    </div>
                                    <div class="col-md-8 col-sm-12">
                                        <h3>Cmai</h3>
                                        <div class="project_crdts"><a
                                                href="https://github.com/ice4prince/Cmai"
                                                target="_blank"><span
                                                    class="fa fa-github">
                                                    Github</span></a> &nbsp;
                                        </div>
                                        <p>In this work, we developed an
                                            innovative
                                            Artificial Intelligence tool, Cmai,
                                            to
                                            address the prediction of binding
                                            between antibodies and antigens that
                                            can
                                            be scaled to high-throughput
                                            sequencing
                                            data.</p>
                                        <hr
                                            style="margin-bottom: 10px; margin-top:0">
                                        <div><a href="./download/cmai/RFoutputs.zip"
                                                target="_blank"
                                                download=""><span
                                                    class="fa fa-download">
                                                    RFoutputs</span></a>
                                        </div>
                                        <p>The intermediate data from preparing
                                            and
                                            processing RoseTTAFold of the
                                            example
                                            data.</p>
                                        <div><a href="./download/cmai/NPY.zip"
                                                target="_blank"
                                                download=""><span
                                                    class="fa fa-download">
                                                    NPY</span></a>
                                        </div>
                                        <p>NPY Description The intermediate data
                                            of
                                            all antigen embeddings of the
                                            example
                                            data.</p>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>

                    <div class="row">
                        <div class="col-md-12 col-sm-12 col-xs-12">
                            <div class="project">
                                <div class="row">
                                    <div class="col-md-4 col-sm-12">
                                        <img src="images/software/spacia_logo.png"
                                            alt="" class="img-responsive">
                                    </div>
                                    <div class="col-md-8 col-sm-12">
                                        <h3>Spacia</h3>
                                        <div class="project_crdts"><a
                                                href="https://github.com/yunguan-wang/Spacia/tree/main"
                                                target="_blank"><span
                                                    class="fa fa-github">
                                                    Github</span></a> &nbsp;
                                        </div>
                                        <p>Here, we introduce a Bayesian
                                            multi-instance learning framework,
                                            Spacia, to detect cell-2-cell
                                            communications from data generated
                                            by
                                            spatially resolved transcriptomics
                                            data,
                                            by uniquely exploiting their spatial
                                            modality.</p>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>

                    <div class="row">
                        <div class="col-md-12 col-sm-12 col-xs-12">
                            <div class="project">
                                <div class="row">
                                    <div class="col-md-4 col-sm-12">
                                        <img src="images/software/toolbox.jpg"
                                            alt="" class="img-responsive">
                                    </div>
                                    <div class="col-md-8 col-sm-12">
                                        <h3>CyTOF DR Playground
                                            and
                                            CyTOF DR Package</h3>
                                        <div class="project_crdts"><a
                                                href="https://dbai.biohpc.swmed.edu/cytof-dr-playground"
                                                target="_blank"><span
                                                    class="fa fa-desktop">
                                                    Web</span></a> &nbsp;<a
                                                href="https://CytofDR.readthedocs.io"
                                                target="_blank"><span
                                                    class="fa fa-desktop">
                                                    Package</span></a> &nbsp;
                                        </div>
                                        <p>Based on our systematic benchmark
                                            study
                                            (Wang et al, 2023, Nature
                                            Communications), we developed a set
                                            of
                                            freely available web resources to
                                            help
                                            users select the best DR methods for
                                            their own datasets, and to aid in
                                            the
                                            development of improved DR
                                            algorithms
                                            tailored to the increasingly popular
                                            CyTOF technique.</p>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>

                    <div class="row">
                        <div class="col-md-12 col-sm-12 col-xs-12">
                            <div class="project">
                                <div class="row">
                                    <div class="col-md-4 col-sm-12">
                                        <img src="images/software/toolbox.jpg"
                                            alt="" class="img-responsive">
                                    </div>
                                    <div class="col-md-8 col-sm-12">
                                        <h3>Benisse</h3>
                                        <div class="project_crdts"><a
                                                href="https://github.com/wooyongc/Benisse"
                                                target="_blank"><span
                                                    class="fa fa-github">
                                                    Github</span></a> &nbsp;
                                        </div>
                                        <p>We developed the Benisse model (BCR
                                            embedding graphical network informed
                                            by
                                            scRNA-Seq) to provide refined
                                            analyses
                                            of BCRs guided by single cell gene
                                            expression. Benisse revealed a
                                            gradient
                                            of B cell activation along BCR
                                            trajectories. We found that BCRs
                                            form a
                                            directed pattern of continuous and
                                            linear evolution to achieve the
                                            highest
                                            antigen targeting efficiency,
                                            compared
                                            with the convergent evolution
                                            pattern of
                                            T cell receptors (TCRs).</p>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>

                    <div class="row">
                        <div class="col-md-12 col-sm-12 col-xs-12">
                            <div class="project">
                                <div class="row">
                                    <div class="col-md-4 col-sm-12">
                                        <img src="images/software/toolbox.jpg"
                                            alt="" class="img-responsive">
                                    </div>
                                    <div class="col-md-8 col-sm-12">
                                        <h3>BepiTBR</h3>
                                        <div class="project_crdts"><a
                                                href="https://dbai.biohpc.swmed.edu/bepitbr/"
                                                target="_blank"><span
                                                    class="fa fa-desktop">
                                                    Web</span></a> &nbsp;<a
                                                href="https://github.com/zzhu33/BepiTBR"
                                                target="_blank"><span
                                                    class="fa fa-github">
                                                    Github</span></a> &nbsp;
                                        </div>
                                        <p>We developed a linear B cell epitope
                                            prediction model, BepiTBR, based on
                                            T-B
                                            reciprocity, and performed
                                            comprehensive
                                            validations. We showed that
                                            explicitly
                                            including the enrichment of putative
                                            CD4+ T-cell epitopes (predicted HLA
                                            class II epitopes) in the model
                                            leads to
                                            significant enhancement in the
                                            prediction of linear B cell
                                            epitopes.
                                        </p>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>

                    <div class="row">
                        <div class="col-md-12 col-sm-12 col-xs-12">
                            <div class="project">
                                <div class="row">
                                    <div class="col-md-4 col-sm-12">
                                        <img src="images/software/toolbox.jpg"
                                            alt="" class="img-responsive">
                                    </div>
                                    <div class="col-md-8 col-sm-12">
                                        <h3>dCLIP</h3>
                                        <div class="project_crdts"><a
                                                href="https://qbrc.swmed.edu/labs/xiaoxie/download/dCLIP_1.7.tar.gz"
                                                target="_blank"
                                                download=""><span
                                                    class="fa fa-download">
                                                    Software</span></a> &nbsp;
                                        </div>
                                        <p>dCLIP is written in Perl for
                                            discovering
                                            differential binding regions in two
                                            CLIP-Seq (HITS-CLIP or PAR-CLIP)
                                            experiments. It is appropriate in
                                            experiments where the common binding
                                            regions that are significantly
                                            enriched
                                            in both conditions tend to have
                                            similar
                                            binding strength and when
                                            researchers
                                            are more interested in the
                                            difference in
                                            binding strength rather than the
                                            binary
                                            event of whether binding site is
                                            common
                                            or not.</p>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>

                    <div class="row">
                        <div class="col-md-12 col-sm-12 col-xs-12">
                            <div class="project">
                                <div class="row">
                                    <div class="col-md-4 col-sm-12">
                                        <img src="images/software/toolbox.jpg"
                                            alt="" class="img-responsive">
                                    </div>
                                    <div class="col-md-8 col-sm-12">
                                        <h3>Netie</h3>
                                        <div class="project_crdts"><a
                                                href="https://github.com/tianshilu/Netie"
                                                target="_blank"><span
                                                    class="fa fa-github">
                                                    Github</span></a> &nbsp;
                                        </div>
                                        <p>We develop a hierarchical Bayesian
                                            model,
                                            named Neoantigen-T cell Interaction
                                            Estimation (netie) to infer the
                                            history
                                            of neoantigen-CD8+ T cell
                                            interactions
                                            in tumors. We showed that tumors
                                            with an
                                            increase in immune selection
                                            pressure
                                            over time, inferred from the
                                            distribution of neoantigens,
                                            demonstrate
                                            an expression signature of
                                            activation of
                                            T cells. We also discovered
                                            exhaustion
                                            of T cell cytotoxic activity post
                                            receiving immunotherapy treatment in
                                            the
                                            tumor clones that newly arise after
                                            treatment.</p>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>

                    <div class="row">
                        <div class="col-md-12 col-sm-12 col-xs-12">
                            <div class="project">
                                <div class="row">
                                    <div class="col-md-4 col-sm-12">
                                        <img src="images/software/toolbox.jpg"
                                            alt="" class="img-responsive">
                                    </div>
                                    <div class="col-md-8 col-sm-12">
                                        <h3>DisHet</h3>
                                        <div class="project_crdts"><a
                                                href="https://dbai.biohpc.swmed.edu/dishet"
                                                target="_blank"><span
                                                    class="fa fa-desktop">
                                                    Web</span></a> &nbsp;<a
                                                href="https://cran.r-project.org/web/packages/DisHet/index.html"
                                                target="_blank"
                                                download=""><span
                                                    class="fa fa-download">
                                                    Software</span></a> &nbsp;
                                        </div>
                                        <p>An R package for estimating the gene
                                            expression levels and component
                                            proportions of the normal, stroma
                                            (immune) and tumor components of
                                            bulk
                                            tumors. The DisHet package also
                                            documents a series of gene
                                            signatures
                                            for tumor infiltrating immune cells,
                                            which are defined with empirical
                                            evidence gained from DisHet analysis
                                            of
                                            35 RCC trio RNA-Seq data.</p>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>

                    <div class="row">
                        <div class="col-md-12 col-sm-12 col-xs-12">
                            <div class="project">
                                <div class="row">
                                    <div class="col-md-4 col-sm-12">
                                        <img src="images/software/toolbox.jpg"
                                            alt="" class="img-responsive">
                                    </div>
                                    <div class="col-md-8 col-sm-12">
                                        <h3>pMTnet</h3>
                                        <div class="project_crdts"><a
                                                href="https://dbai.biohpc.swmed.edu/pmtnet/"
                                                target="_blank"><span
                                                    class="fa fa-desktop">
                                                    Web</span></a> &nbsp;<a
                                                href="https://github.com/tianshilu/pMTnet"
                                                target="_blank"><span
                                                    class="fa fa-github">
                                                    Github</span></a> &nbsp;
                                        </div>
                                        <p>A deep learning neural network model
                                            for
                                            predicting TCR binding specificity
                                            of
                                            peptides presented by class I HLA
                                            alleles</p>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>

                    <div class="row">
                        <div class="col-md-12 col-sm-12 col-xs-12">
                            <div class="project">
                                <div class="row">
                                    <div class="col-md-4 col-sm-12">
                                        <img src="images/software/toolbox.jpg"
                                            alt="" class="img-responsive">
                                    </div>
                                    <div class="col-md-8 col-sm-12">
                                        <h3>QBRC mutation
                                            calling
                                            pipeline</h3>
                                        <div class="project_crdts"><a
                                                href="https://github.com/Somatic-pipeline/Somatic-pipeline"
                                                target="_blank"><span
                                                    class="fa fa-github">
                                                    Github</span></a> &nbsp;
                                        </div>
                                        <p>The QBRC mutation calling pipeline is
                                            a
                                            flexible and comprehensive pipeline
                                            for
                                            mutation calling that has glued
                                            together
                                            a lot of commonly used software and
                                            data
                                            processing steps for mutation
                                            calling.
                                            The mutation calling software
                                            include:
                                            sambamba, speedseq, varscan,
                                            shimmer,
                                            strelka, manta, lofreq_tar. It
                                            identifies somatic and germline
                                            variants
                                            from whole exome sequencing (WXS),
                                            RNA
                                            sequencing and deep sequencing data.
                                            It
                                            can be used for human, PDX, and
                                            mouse
                                            data (fastq files or bam files as
                                            input).</p>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>

                    <div class="row">
                        <div class="col-md-12 col-sm-12 col-xs-12">
                            <div class="project">
                                <div class="row">
                                    <div class="col-md-4 col-sm-12">
                                        <img src="images/software/toolbox.jpg"
                                            alt="" class="img-responsive">
                                    </div>
                                    <div class="col-md-8 col-sm-12">
                                        <h3>QBRC neoantigen
                                            calling
                                            pipeline</h3>
                                        <div class="project_crdts"><a
                                                href="https://github.com/Neoantigen-pipeline/Neoantigen-pipeline"
                                                target="_blank"><span
                                                    class="fa fa-github">
                                                    Github</span></a> &nbsp;
                                        </div>
                                        <p>The QBRC neoantigen calling pipeline
                                            is a
                                            comprehensive and user-friendly
                                            neoantigen calling pipeline for
                                            human
                                            genomics samples. It needs the
                                            somatic
                                            mutation calling results of the QBRC
                                            mutation calling pipeline, the
                                            tumor/normal exome-seq data for HLA
                                            typing, and optionally RNA-seq data
                                            for
                                            filtering neoantigens called from
                                            the
                                            exome-seq data. It profiles both MHC
                                            I
                                            and II-binding neoantigens. The
                                            calculation of CSiN (Cauchy-Schwarz
                                            index of Neoantigens), which
                                            describes
                                            neoantigen clonal balance, is
                                            embedded
                                            in the pipeline.</p>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>

                    <div class="row">
                        <div class="col-md-12 col-sm-12 col-xs-12">
                            <div class="project">
                                <div class="row">
                                    <div class="col-md-4 col-sm-12">
                                        <img src="images/software/toolbox.jpg"
                                            alt="" class="img-responsive">
                                    </div>
                                    <div class="col-md-8 col-sm-12">
                                        <h3>SCINA</h3>
                                        <div class="project_crdts"><a
                                                href="https://dbai.biohpc.swmed.edu/scina/"
                                                target="_blank"><span
                                                    class="fa fa-desktop">
                                                    Web</span></a> &nbsp;<a
                                                href="https://github.com/jcao89757/SCINA"
                                                target="_blank"><span
                                                    class="fa fa-github">
                                                    Github</span></a> &nbsp;
                                        </div>
                                        <p>SCINA is a semi-supervised algorithm
                                            for
                                            identification of cell types in
                                            single
                                            cell profiling data. It
                                            automatically
                                            exploits prior knowledge of cell
                                            type-specific signatures as a form
                                            of
                                            supervision. It also works for
                                            disease
                                            subtyping at the patient level, or
                                            other
                                            scenarios where data of similar
                                            format
                                            are available.</p>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>

                    <div class="row">
                        <div class="col-md-12 col-sm-12 col-xs-12">
                            <div class="project">
                                <div class="row">
                                    <div class="col-md-4 col-sm-12">
                                        <img src="images/software/toolbox.jpg"
                                            alt="" class="img-responsive">
                                    </div>
                                    <div class="col-md-8 col-sm-12">
                                        <h3>SClineager</h3>
                                        <div class="project_crdts"><a
                                                href="https://github.com/inmybrain/SClineager"
                                                target="_blank"><span
                                                    class="fa fa-github">
                                                    Github</span></a> &nbsp;
                                        </div>
                                        <p>We introduce SClineager that infers
                                            accurate evolutionary lineages from
                                            scRNA-seq data by borrowing
                                            information
                                            from related cells to overcome
                                            expressional drop-outs. SClineager
                                            provides a powerful tool that can be
                                            applied to scRNA-Seq data to
                                            decipher
                                            the lineage histories of cells, and
                                            which could address a missing
                                            opportunity to reveal valuable
                                            information from the large amounts
                                            of
                                            existing scRNA-Seq data.</p>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>

                    <div class="row">
                        <div class="col-md-12 col-sm-12 col-xs-12">
                            <div class="project">
                                <div class="row">
                                    <div class="col-md-4 col-sm-12">
                                        <img src="images/software/toolbox.jpg"
                                            alt="" class="img-responsive">
                                    </div>
                                    <div class="col-md-8 col-sm-12">
                                        <h3>Sprod</h3>
                                        <div class="project_crdts"><a
                                                href="https://github.com/yunguan-wang/SPROD"
                                                target="_blank"><span
                                                    class="fa fa-github">
                                                    Github</span></a> &nbsp;
                                        </div>
                                        <p>We developed a mathematical model,
                                            named
                                            Sprod, to impute accurate ST gene
                                            expression based on latent space and
                                            graph learning of matched location
                                            and
                                            imaging data. We comprehensively
                                            validated Sprod and demonstrated its
                                            advantages over prior methods for
                                            removing drop-outs in scRNA-seq
                                            data.
                                        </p>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>

                    <div class="row">
                        <div class="col-md-12 col-sm-12 col-xs-12">
                            <div class="project">
                                <div class="row">
                                    <div class="col-md-4 col-sm-12">
                                        <img src="images/software/toolbox.jpg"
                                            alt="" class="img-responsive">
                                    </div>
                                    <div class="col-md-8 col-sm-12">
                                        <h3>Tessa</h3>
                                        <div class="project_crdts"><a
                                                href="https://github.com/jcao89757/tessa"
                                                target="_blank"><span
                                                    class="fa fa-github">
                                                    Github</span></a> &nbsp;
                                        </div>
                                        <p>Tessa is a Bayesian model to
                                            intergrate T
                                            cell receptor (TCR) sequence
                                            profiling
                                            with transcriptomes of T cells.
                                            Enabled
                                            by the recently developed single
                                            cell
                                            sequencing techniques, which provide
                                            both TCR sequences and RNA sequences
                                            of
                                            each T cell concurrently, Tessa maps
                                            the
                                            functional landscape of the TCR
                                            repertoire, and generates insights
                                            into
                                            understanding human immune response
                                            to
                                            diseases.</p>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>

                    <div class="row">
                        <div class="col-md-12 col-sm-12 col-xs-12">
                            <div class="project">
                                <div class="row">
                                    <div class="col-md-4 col-sm-12">
                                        <img src="images/software/toolbox.jpg"
                                            alt="" class="img-responsive">
                                    </div>
                                    <div class="col-md-8 col-sm-12">
                                        <h3>Tissue Positioning
                                            System (TPS)</h3>
                                        <div class="project_crdts"><a
                                                href="https://github.com/yunguan-wang/liver_zone_segmentation"
                                                target="_blank"><span
                                                    class="fa fa-github">
                                                    Github</span></a> &nbsp;
                                        </div>
                                        <p>TPS is a quantitative, unsupervised
                                            algorithm for zonated expression
                                            pattern
                                            detection in immunofluorescence
                                            images.
                                            It is developed for evaluating
                                            zonated
                                            protein expression in hepatocytes.
                                            We
                                            welcome suggestions for other
                                            potential
                                            use cases. For current hepatocyte
                                            application, TPS requires an input
                                            IF
                                            image with DAPI channel for nuclei
                                            and
                                            GS channel for central veins.</p>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
            </section>
        </main>
        <!--End Main Content Area-->

        <!--Footer-->
        <div class="bg">
            <div class="container" style="padding: 30px 30px">
                <footer id="landing-footer" class="clearfix">
                    <div class="clearfix">
                        <p>
                            <a>2025 &nbsp; Â© &nbsp;</a>
                            <a href="index.html">Tao
                                Wang Lab &nbsp;|&nbsp;</a>
                            <a
                                href="https://www.mdanderson.org/research/departments-labs-institutes/departments-divisions/bioinformatics-and-computational-biology.html">Bioinformatics
                                and
                                Computational Biology Department &nbsp; |
                                &nbsp;</a>
                            <a href="https://www.mdanderson.org/"> MD
                                Anderson Cancer Center &nbsp; |
                                &nbsp;</a>
                            <a href="" style="vertical-align: middle"><i
                                    class="fa fa-twitter fa-2x"></i>
                        </p>
                    </div>
                </footer>
            </div>
        </div>
        <!--End of Footer-->

    </div>

    <!-- Include JavaScript resources -->
    <script src="js/jquery.1.8.3.min.js"></script>
    <script src="js/wow.min.js"></script>
    <script src="js/featherlight.min.js"></script>
    <script src="js/featherlight.gallery.min.js"></script>
    <script src="js/jquery.enllax.min.js"></script>
    <script src="js/jquery.scrollUp.min.js"></script>
    <script src="js/jquery.easing.min.js"></script>
    <!-- <script src="js/jquery.stickyNavbar.min.js"></script> -->
    <script src="js/jquery.waypoints.min.js"></script>
    <script src="js/images-loaded.min.js"></script>
    <script src="js/lightbox.min.js"></script>
    <script src="js/site.js"></script>
    <script src="js/custom.js"></script>
    <script type='text/javascript' src="js/collapsibleclick.js"></script>
</body>

</html>